homework02.py

```diff
@@ -0,0 +1,17 @@
+text = input("Write word or number:")
+if text.isdigit():
+if int(text) % 2 == 0:
```

Comment:
The nested `if` condition should have an indentation of 4 spaces (1 tab). Your code raises an error:
<img width="674" alt="image" src="https://user-images.githubusercontent.com/18496465/206926353-79b595c8-8c7a-4772-a5bd-090eacb404dd.png">